Gemfile.lock

```diff
@@ -15,36 +15,62 @@ GEM
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
+    addressable (2.8.1)
+      public_suffix (>= 2.0.2, < 6.0)
     colorize (0.8.1)
     concurrent-ruby (1.1.10)
-    connection_pool (2.2.5)
+    connection_pool (2.3.0)
+    declarative (0.0.20)
     diff-lcs (1.5.0)
     domain_name (0.5.20190701)
       unf (>= 0.0.5, < 1.0.0)
     faraday (0.17.6)
       multipart-post (>= 1.2, < 3)
-    google_drive (0.3.10)
-      nokogiri (>= 1.4.4, != 1.5.2, != 1.5.1)
-      oauth (>= 0.3.6)
-      oauth2 (>= 0.5.0)
-    hashie (5.0.0)
+    google-apis-core (0.10.0)
+      addressable (~> 2.5, >= 2.5.1)
+      googleauth (>= 0.16.2, < 2.a)
+      httpclient (>= 2.8.1, < 3.a)
+      mini_mime (~> 1.0)
+      representable (~> 3.0)
+      retriable (>= 2.0, < 4.a)
+      rexml
+      webrick
+    google-apis-drive_v3 (0.33.0)
+      google-apis-core (>= 0.9.1, < 2.a)
+    google-apis-sheets_v4 (0.21.0)
+      google-apis-core (>= 0.9.1, < 2.a)
+    google_drive (3.0.7)
```

Comment:
This jumped from 0.3.10 to 3.0.7.

The gem's documentation suggests that will not work: https://github.com/gimite/google-drive-ruby/blob/master/MIGRATING.md

Reply:
Actually, the documentation says the old version of the gem doesn't work at all because it uses an API that has been deprecated and shut down.

My theory is that the at some point we used a Google Drive spreadsheet as a storage layer but this broke a long long time ago, which doesn't matter because we also don't use it anymore.

Reply:
I think there are a few places we've used the google spreadsheets for things; but rarely. Someone on the USC would know better.